x-pack/plugins/index_lifecycle_management/public/application/sections/policy_list/policy_list.container.tsx

```diff
@@ -30,47 +25,43 @@ export const PolicyList: React.FunctionComponent = () => {
 
   if (isLoading) {
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
-        <EuiEmptyPrompt
-          title={<EuiLoadingSpinner size="xl" />}
-          body={
-            <FormattedMessage
-              id="xpack.indexLifecycleMgmt.policyTable.policiesLoading"
-              defaultMessage="Loading policies..."
-            />
-          }
-        />
-      </EuiPageContent>
+      <EuiPageTemplate.EmptyPrompt
+        title={<EuiLoadingSpinner size="xl" />}
+        body={
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.policyTable.policiesLoading"
+            defaultMessage="Loading policies..."
+          />
+        }
+      />
     );
   }
   if (error) {
     const { statusCode, message } = error ? error : { statusCode: '', message: '' };
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
-        <EuiEmptyPrompt
-          title={
-            <h2>
-              <FormattedMessage
-                id="xpack.indexLifecycleMgmt.policyTable.policiesLoadingFailedTitle"
-                defaultMessage="Unable to load existing lifecycle policies"
-              />
-            </h2>
-          }
-          body={
-            <p>
-              {message} ({statusCode})
-            </p>
-          }
-          actions={
-            <EuiButton onClick={resendRequest} iconType="refresh" color="danger">
-              <FormattedMessage
-                id="xpack.indexLifecycleMgmt.policyTable.policiesReloadButton"
-                defaultMessage="Try again"
-              />
-            </EuiButton>
-          }
-        />
-      </EuiPageContent>
+      <EuiPageTemplate.EmptyPrompt
```

Comment:
Please add prop `color="danger"`

```suggestion
      <EuiPageTemplate.EmptyPrompt color="danger"
```

Current
<img width="628" alt="Screenshot 2023-08-16 at 3 32 06 PM" src="https://github.com/elastic/kibana/assets/5226211/2a5e3fcb-6df4-43a5-8b86-9866ba174f35">

Expected
<img width="632" alt="Screenshot 2023-08-16 at 3 32 20 PM" src="https://github.com/elastic/kibana/assets/5226211/27a695e4-9f2d-4317-9e52-5b371b95de98">